server/src/main/java/com/google/coffeehouse/common/StorageHandler.java

```diff
@@ -102,61 +224,110 @@ public static String getClubQuery(DatabaseClient dbClient, String clubId) {
       try (ResultSet resultSet = dbClient.singleUse().executeQuery(statement)) {
         while (resultSet.next()) {
           bookId = resultSet.getString("bookId");
+          name = resultSet.getString("name");
           description = resultSet.getString("description");
           ownerId = resultSet.getString("ownerId");
-          name = resultSet.getString("name");
         }
-        clubInfo = String.format(
-                          "Club ID: %s || Book ID: %s || Description: %s || Name: %s || Owner ID: %s\n",
-                          clubId, bookId, description, name, ownerId);
       }
+      Book book = getBookQuery(dbClient, bookId);
+      Club.Builder clubBuilder = Club.newBuilder(name, book);
+      club = clubBuilder
+                        .setDescription(description)
```

Comment:
Should the `ownerId` be set here?

Reply:
This is on me, the ownerId field is currently not in any pushed Club.java code. If Lina adds a TODO comment here I'll cover this change.

Reply:
I'll make sure to add the ID fields as TODOs for the rest of the file too -- sorry about that!